practice-areas/project-management/mods-and-extensions.md

```diff
@@ -6,32 +6,52 @@ title: Contractual Modification (MOD) or Extension
 
 ## What is the difference between a contract Mod and an Extension?
 
-Contract Modifications (Mod) are required for any change to the original document, e.g. adjustments to the following:
+1/ Contract Modifications (Mods) are required for any change to the project contract or PWS, e.g. adjustments to the following:
 
--   key personnel
--   scope
--   deliverables
--   contract amount
+A/ Key personnel – Key Personnel are defined as "...personnel… considered to be essential to the work being performed under [a] contract." In many cases, Key Personnel are explicitly named in the contact, and whenever this is the case, changes to Key Personnel must be denoted with a Mod.
```

Comment:
@hhartingrex looks like the remark link doesn't like the `[a]`. That can be escaped if it cannot be re-written.